### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Ambrosegithub**
- Create a new branch.
- add the js folder and js file.
- add HTML dynamically on the page
- receive contributions from partner

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -683,6 +694,118 @@ footer h6 {
...
+  flex-direction: column;
+  text-align: center;
+  transition: 1s;
+  width: 85%;
+  color: #1a2236;
+  right: 50%;
+  margin-bottom: 12px;
+  z-index: 999;
+  background-color: #fff;
+}
```
Comment (vechicin) [inline style.css]: Your design doesn't quite match the template design.

Your design:

Screen Shot 2022-06-20 at 10 09 51

The template design:

Screen Shot 2022-06-20 at 10 11 04

Please fix it.

Additionally, the link for the `About` section is not redirecting to the `About` section, please fix that as well.

File: style.css [hunk 2]
```diff
@@ -19,6 +19,8 @@ body {
   display: flex;
   width: 100%;
   height: 104px;
+  justify-content: space-between;
+  align-items: center;
 }
```
Comment (vechicin) [inline style.css:24]: I cannot see the navbar in the desktop section anymore.

Screen Shot 2022-06-20 at 10 15 59

Please fix it.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Ambrosegithub ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good attempt at implementing the mobile menu

## Required Changes ♻️

- [ ] For this project, since you're using JavaScript, you need to install ESLint. Please do so.

Screen Shot 2022-06-20 at 10 18 06

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kandy-Peter — APPROVED** [review]
Hi @Ambrosegithub ,

### APPROVED 🟢 

image gif

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**